aquarius/app/util.py

```diff
@@ -70,21 +68,14 @@ def get_metadata_from_services(services):
             return service["attributes"]
 
 
-def reorder_services_list(services):
-    service_dict = OrderedDict([(s["type"], s) for s in services])
-    result = [service_dict.pop("metadata")]
-    result.extend([s for t, s in service_dict.items()])
-
-    return result
-
-
 def init_new_ddo(data, timestamp):
     _record = copy.deepcopy(data)
     _record["created"] = format_timestamp(
         datetime.fromtimestamp(timestamp).strftime(DATETIME_FORMAT)
     )
     _record["updated"] = _record["created"]
 
+    # TODO: still needed?
```

Comment:
The [v4 DDO spec](https://github.com/oceanprotocol/docs/blob/feature/ddo_v4/content/concepts/did-ddo.md) doesn't mention `accessWhiteList` so I think we should remove this and all other mentions of `accessWhiteList`.

Just to be sure, I searched through historical commits and issues.

Issue https://github.com/oceanprotocol/multi-repo-issue/issues/50 was going to use the `allowWhiteList` property, but it was closed without changes.
This code was added in PR #266 but even at that time it seems to be a vestige of an older design.  In the same PR, multiple unit tests relating to `accessWhiteList` were removed. 
* Added: https://github.com/oceanprotocol/aquarius/pull/266/files#diff-a16473cce6b6252e953d1ae0ac253984c03eff6fa90ede50ed02cdda6ed4e9d7R96-R102
* Removed unit tests: https://github.com/oceanprotocol/aquarius/pull/266/files#diff-3451cc474d5698222168f66a293339a6bd296ee2c56f56409866bbae99917a79L409-L482